src/main/java/seedu/address/model/person/JerseyNumber.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.address.logic.parser.ParserUtil.UNSPECIFIED_FIELD;
+
+/**
+ * Represents a Player's jersey number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidJerseyNumber(String)}
+ */
+public class JerseyNumber {
+
+    public static final String MESSAGE_JERSEY_NUMBER_CONSTRAINTS =
+            "Player's jersey number should be an integer from 0 - 99.";
+
+
+    public static final String RATING_VALIDATION_REGEX = "[0-9]|[1-8][0-9]|9[0-9]";
```

Comment:
This regex tested? Because currently using regxr.com, i'm able to input values such as 999.